models/transaction_test.go

```diff
@@ -13,10 +14,36 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func createTestEvent(t *testing.T) (cadence.Event, *gethTypes.Transaction, *types.Result) {
+//go:embed fixtures/evm_tx.bin
+var evmTxBinary string
```

Comment:
just for the sake of keeping the same terminology rename to: `transaction_call.bin`

Reply:
Good point :+1: Updated in https://github.com/onflow/flow-evm-gateway/pull/132/commits/579651ca342720c7aed8eceb5496dafbc03005af